projects/m2/016-reverse-lookup/js/index.js

```diff
@@ -0,0 +1,29 @@
+function reverseLookup(dictionary, value){
+    const keys = [];
+
+    Object.keys( dictionary ).forEach( key => {

```

Comment:
In this case, **why** did you use the `forEach` method instead of the **map**? 
What is the difference between the two and why **in this case** could the `map` be **recommended**?

Reply:
Hi, the use of map is recommended in case you want to obtain an array directly by processing the original.

However, I preferred the use of filter because the array returned by map is of the same length as the original, while filter returns one containing only the values ​​that meet the indicated condition.

Is it ok?